portfolio/src/main/webapp/index.html

```diff
@@ -1,18 +1,174 @@
 <!DOCTYPE html>
 <html>
   <head>
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
     <meta charset="UTF-8">
     <title>My Portfolio</title>
+
+    <link href="https://fonts.googleapis.com/css2?family=Lato:ital@1&display=swap" rel="stylesheet">
+    
     <link rel="stylesheet" href="style.css">
     <script src="script.js"></script>
   </head>
   <body>
-    <div id="content">
-      <h1>My Portfolio</h1>
-      <p>This is my portfolio.</p>
-      <p>Click here to get a random message:</p>
-      <button onclick="addRandomGreeting()">Hello</button>
-      <div id="greeting-container"></div>
+        <nav> 
+            <span>Navbar</span>
+            <a href="#about-content">About</a>
+            <a href="#gallery">Gallery</a>
+            <a href="#box">Color Picker</a>
+        </nav>
+
+    <div class="header"> 
+        <div class="header-text"> 
+            <h1>Sofia Montana</h1>
+        </div>
     </div>
+
+    <div id="about-content"> 
```

Comment:
Use a consistent style when choosing between "id" or "class". "header" above is a class while this is an id. I'd go with just making everything a class unless it needs to be an id to be used with the JavaScript (like "box" and "color").